### PR TITLE
Update RBAC ContentGuard commands for 3.17

### PR DESCRIPTION
**Pull request by gerrod3**
[noissue]

### CODE REVIEW

File: pulpcore/cli/core/content_guard.py [hunk 1]
```diff
@@ -58,13 +58,13 @@ def rbac(ctx: click.Context, pulp_ctx: PulpContext) -> None:
...
     "groups",
-    help=_("Group to remove download permission from. Can be specified multiple times."),
+    help=_("Group to add download role to. Can be specified multiple times."),
     multiple=True,
 )
 @click.option(
     "--user",
     "users",
-    help=_("User to remove download permission from. Can be specified multiple times."),
+    help=_("User to add download role to. Can be specified multiple times."),
```
Comment (mdellweg) [inline pulpcore/cli/core/content_guard.py:67]: :+1:

File: tests/scripts/pulpcore/test_content_guards.sh [hunk 2]
```diff
@@ -20,7 +20,9 @@ expect_succ pulp content-guard rbac show --name "cli_test_guard"
 
 expect_succ pulp group create --name "cli_test_group"
 expect_succ pulp content-guard rbac assign --name "cli_test_guard" --group "cli_test_group"
+expect_succ pulp content-guard rbac show --name "cli_test_guard"
 test "$(echo "$OUTPUT" | jq -r '.groups' | jq -r length)" -eq "1"
 expect_succ pulp content-guard rbac remove --name "cli_test_guard" --user "admin" --group "cli_test_group"
+expect_succ pulp content-guard rbac show --name "cli_test_guard"
```
Comment (mdellweg) [inline tests/scripts/pulpcore/test_content_guards.sh:26]: The groups and users listed here are determined by the permission, right? This is a little bit dangerous, if the admin changes the access policy to use a different permission, or a non trivial condition. Also not a problem to solve here.

**Review by mdellweg — APPROVED** [review]
assuming the merge of https://github.com/pulp/pulpcore/pull/1676 this looks good to me

File: tests/scripts/pulpcore/test_content_guards.sh (hunk 2 above)
Comment (gerrod3) [reply tests/scripts/pulpcore/test_content_guards.sh:26]: Yes they are, something to think about.

**Pull request merged**